samples/connectivity/callnotification/README.md

```diff
@@ -0,0 +1,3 @@
+# CallNotificationSample samples
+
+// Sample for posting call and in call notifications
```

Comment:
Could you provide a little bit better README. No need to be too detailed but at least a small explanation and a link to docs